docs/config-file-package-configuration.yml.md

```diff
@@ -0,0 +1,101 @@
+# The package configuration file (Experimental)
```

Comment:
See other docs we use capitals in section headers so this should be `# The Package Configuration File`